### ISSUE TITLE
Making a release with current version of code

### ISSUE THREAD

**Issue by dogoepp** [resibots/dynamixel_control_hw#10]
Hi, what about we tag current version of the code with something like v0.1.0 since we changed the library dependency ?

**Comment by dogoepp** [comment]
What about we actually change the version number ? We now have velocity control and are about to have joint limits.

What do you think @costashatz ?

**Comment by costashatz** [comment]
Yes it might be a good idea! Once the joint limits are working we should change the version number and make a release! :smile:
